package.json

```diff
@@ -63,31 +67,38 @@
     ]
   },
   "devDependencies": {
+    "@4c/babel-preset-4catalyzer": "^4.1.1",
     "@4c/jest-preset": "^1.3.0",
+    "@4c/semantic-release-config": "^1.0.2",
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.2.2",
-    "@4c/babel-preset-4catalyzer": "^4.1.1",
-    "babel-core": "^7.0.0-bridge.0",
+    "@babel/preset-typescript": "^7.1.0",
     "@types/node": "*",
+    "@typescript-eslint/eslint-plugin": "^1.1.1",
+    "babel-core": "^7.0.0-bridge.0",
     "eslint": "^5.12.0",
+    "eslint-config-4catalyzer": "^0.7.0",
+    "eslint-config-4catalyzer-jest": "^1.1.0",
+    "eslint-config-4catalyzer-typescript": "^0.10.1",
     "eslint-config-prettier": "^4.0.0",
     "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-jest": "^22.1.2",
-    "eslint-config-4catalyzer-jest": "^1.1.0",
-    "eslint-config-4catalyzer": "^0.7.0",
-    "typescript": "^3.2.2",
-    "@babel/preset-typescript": "^7.1.0",
-    "eslint-config-4catalyzer-typescript": "^0.10.1",
-    "@typescript-eslint/eslint-plugin": "^1.1.1",
+    "eslint-plugin-prettier": "^3.0.1",
+    "husky": "^1.1.2",
     "import-sort-cli": "^5.2.0",
     "import-sort-style-4catalyzer": "^4.3.0",
-    "@4c/semantic-release-config": "^1.0.2",
-    "semantic-release": "^15.5.0",
-    "travis-deploy-once": "^5.0.0",
-    "husky": "^1.1.2",
+    "jest": "^24.0.0",
     "lint-staged": "^8.1.0",
     "prettier": "^1.15.3",
-    "jest": "^24.0.0"
+    "semantic-release": "^15.5.0",
+    "travis-deploy-once": "^5.0.0",
+    "typescript": "^3.2.2"
+  },
+  "dependencies": {
+    "@types/fetch-mock": "^7.2.2",
+    "@types/jest": "^23.3.13",
+    "@types/selenium-webdriver": "^3.0.14",
```

Comment:
Types should be in dev deps right?

Reply:
no, because otherwise someone installing this package will not have typings for e.g. `sandbox.fetchMock` unless they explicitly install `@types/fetch-mock`.

Reply:
hmm, that isn't my experience...maybe i've just gotten lucky.

Reply:
Yeah probably. I mean, think about it right? They will just not be installed.

Reply:
I guess i assumed typescript automatically pulled the types...they seem to do that in vscode in some cases anyway

Reply:
yes but that's an editor thing. will not happen in ci or whatever